src/modules/configurator/components/ConfigPropertySkeleton.tsx

```diff
@@ -0,0 +1,8 @@
+export function ConfigPropertySkeleton() {
+  return (
+    <div className="flex items-center py-2 xs:py-4 sm:py-5">
+      <div className="w-12 2xl:w-14 h-12 2xl:h-14 bg-light-gray-background-color rounded-[50%] animate-pulse" />
```

Comment:
You can use rounded-full for rounded-[50%] because 50% is the maximum radius so this is equal to rounded-full.

Reply:
Done.